core/transfer/src/test/java/org/eclipse/dataspaceconnector/transfer/core/transfer/TransferProcessManagerImplTest.java

```diff
@@ -613,4 +628,25 @@ public static Builder newInstance() {
             }
         }
     }
+
+    private static class TokenTestProvisionResource extends TestProvisionedDataDestinationResource {
+        public TokenTestProvisionResource(String resourceName, String id) {
+            super(resourceName, id);
+            this.hasToken = true;
+        }
+    }
+
+    private class TestToken implements SecretToken {
```

Comment:
a `mock` instance would replace well this dummy class

Reply:
The problem is it needs to be serialized to the vault, so a mock will throw an exception when the ObjectMapper attempts the write.

Reply:
got it, thanks